tests/saw/spec/handshake/handshake.saw

```diff
@@ -75,23 +75,23 @@ let prove_handshake_io_lowlevel = do {
 
 let prove_state_machine = do {
     print "Checking proof that the TLS1.2 RFC simulates our Cryptol s2n spec";
-    prove_print z3 {{ tls12rfcSimulatesS2N_alt `{17} }};
+    prove_print abc {{ tls12rfcSimulatesS2N `{16} }};
```

Comment:
Are you switching to abc as part of reverting changes, or because abc is the faster solver for this case?

Reply:
Part of reverting changes. After reverting back to tls12rfcSimulatesS2N the test failed with z3.